### DISCUSSION TITLE
VOD support

### DISCUSSION THREAD

**Discussion by tracure1337** [streamlink/streamlink-twitch-gui#838] [Q&A] [answered]
Hi and thanks in advance for helping me out here.

This is an awesome app. Everything works but I have one big issues which might be just stupid me.

a) The search does not show results for streams that have ended correct?

I tried searching for "Vim Conference" and nothing appears

b) How can I just enter a link like https://www.twitch.tv/videos/1190279834 and play it?

I also tried searching for the id but no dice.

c) When I get to the view of a specific channel I can't find any way to select or see already finished streams. 

Thanks Thanks  Thanks :-)

**Comment by bastimeyer** [comment, marked as answer]
VODs are unsupported in the Twitch GUI. See #81 why.
Use the Streamlink CLI and set the `--player-passthrough=hls` parameter if you want to watch VODs.

**Reply by tracure1337** [reply]
Thanks so much!

**Reply by tracure1337** [reply]
streamlink --player-passthrough=hls https://www.twitch.tv/videos/1190279834 720p60
works but it starts VLC player. How can I change the chosen player? Or force it to set the one set in the GUI?
I had a look at the wiki but ```--launch``` didn't help.

**Reply by bastimeyer** [reply]
https://streamlink.github.io/cli.html
